src/common/htmlFileReader.test.ts

```diff
@@ -9,7 +9,7 @@ describe("Html File Reader", () => {
         const file = new File([blob], "test.txt");
 
         const actualText = await HtmlFileReader.readAsText(file);
-        expect(actualText).toEqual(expectedText);
+        expect(actualText.content).toEqual(expectedText);
```

Comment:
What causes this test to require this change?

Reply:
I am now having the reader return both the contents of the file and the file information. This is because the V1 project format doesn't include any information about the file itself. I'm using that info to get the name and the path, which is necessary for creating a V2 project as well as saving the assets in the proper place